src/playground-helper.ts

```diff
@@ -0,0 +1,31 @@
+import {compressToEncodedURIComponent} from 'lz-string'
+
+function unindent(value: string) {
+  // remove white spaces first, to save a few bytes.
+  // testing-playground will reformat on load any ways.
+  return value.replace(/[ \t]*[\n][ \t]*/g, '\n')
+}
+
+function encode(value: string) {
+  return compressToEncodedURIComponent(unindent(value))
+}
+
+function getPlaygroundUrl(element: Element | null, logErrors = true) {
+  if (!element || !('innerHTML' in element)) {
+    if (logErrors) {
+      console.log(`The element you're providing isn't a valid DOM element.`)
```

Comment:
I'm a bit uncomfortable with logging i.e. side-effects in a `get*` methods. Especially when it's about logging errors. Don't we usually want to throw errors?

Reply:
Makes sense :-) I've changed it to throw errors. The error message is logged (same as before) in `logTestingPlaygroundURL`, but ignored in `getElementError`.